source/fundamentals/monitoring/cluster-monitoring.txt

```diff
@@ -165,10 +165,10 @@ one of the following possible values:
    * - ``RSArbiter``
      - Arbiter instance
    * - ``RSOther``
-     - See the `RSGhost specification <https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#rsghost-and-rsother>`_
+     - See the `RSOther specification <https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#rsghost-and-rsother>`_
```

Comment:
Suggestion:

I think "RSGhost and RSOther" could be a better title since that's the name of the section and it contains information about both.

I think the anonymous link syntax "__" could be more consistent with the rest of our documentation links. I think the "_" actually aliases this link twice since it's the same target.


```suggestion
     - See the `RSGhost and RSOther specification <https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#rsghost-and-rsother>`__
```

Applies to both instances.

Reply:
Good catch, I didn't notice the single underscore here. Updated